tinyagentos/routes/settings.py

```
@@ -1075,31 +1077,88 @@ async def set_update_channel(request: Request, body: UpdateChannel):
+
+
+async def _probe_taosmd(request: Request, url: str) -> bool:
+    """Probe the taOSmd /health endpoint.  Returns True if reachable."""
+    try:
+        client = request.app.state.http_client
+        resp = await client.get(
+            f"{url}/health",
+            timeout=httpx.Timeout(3.0, connect=2.0),
+        )
+        return resp.status_code == 200
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)